content/cloud/changelog/cloud-config.md

```diff
@@ -0,0 +1,23 @@
+title = "Fermyon Cloud Variables Configuration"
+template = "changelog_item"
+date = "2022-06-13T07:00:00Z"
```

Comment:
This is set to publish as 3am EDT on the 13th. Is that intentional?

Reply:
It points to last year. So will be published as soon as it is merged.

Reply:
Good catch on the year!  I have it set to tonight for now to make sure our 1.3 release completes before it goes live.